projects/openapi-utilities/src/openapi3/traverser.ts

```diff
@@ -41,6 +27,97 @@ const isObject = (value: any) => {
   return typeof value === 'object' && !Array.isArray(value) && value !== null;
 };
 
+export function getRaw<T extends V3FactType>(
+  spec: OpenAPIV3.Document,
+  fact: OpenApiV3TraverserFact<T>
+): FactRawItem<T> {
+  return jsonPointer.get(spec, fact.location.jsonPath);
+}
+
+export function getLocation<T extends V3FactType>(
```

Comment:
Because we are responsible for generating the JSON paths, we can rely on stable JSON path generation here. For this, we can encapsulate version differences between V2/V3 / other OAS versions. We can infer basically everything from JSON paths